### PR TITLE
Astro 3/5 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

**Comment by lzdanski** [comment]
related - https://github.com/astronomer/docs/pull/3502

File: astro/release-notes.md [hunk 1]
```diff
@@ -21,6 +21,34 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+Custom Deployment roles are a new way to define granular permissions for Astro users. For the first time, you can set a user's permissions at the Deployment level and define which specific parts of a Deployment they can access or modify. Use custom Deployment roles to have users collaborate in the same Workspace with only the minimum permissions they require. See [Customize Deployment roles](https://docs.astronomer.io/astro/customize-deployment-roles) for more information.
+
+### New Deployment registry cache to improve resiliency
+
+Deployments now include a cache of Astronomer's image registry that stores the current Astro Runtime image for your Deployment. Because Deployments now always have access to their running image, image registry outages should no longer result in failed DAG runs.
+
+### Additional improvements
+
+- Removed nonfunctional network usage per Pod metrics from the Deployment **Analytics** page.
+- The deprecation date for Deployment API keys has been changed from April 1, 2024 to June 1, 2024.
```
Comment (lzdanski) [inline astro/release-notes.md:44]:
```suggestion
- The deprecation date for Deployment API keys has been changed to June 1, 2024.
```

I double checked our docs and looks like we had "april 1-ish", but never socialized/externalized that date. So let's just keep this confirmed date.

File: astro/release-notes.md [hunk 2]
```diff
@@ -21,6 +21,34 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## March 5, 2024
+
+### Export data from reporting dashboards
+
+You can now export data from reporting dashboards in the format of your choice. Exports can be triggered on a regular schedule or as an alert when specific criteria are met in your data. Export reporting data to share with other team members or to keep a record of key performance indicators. See [Export reporting data](https://docs.astronomer.io/astro/reporting-dash-exports) for more information.
```
Comment (lzdanski) [inline astro/release-notes.md:28]:
```suggestion
You can also export data from reporting dashboards in the format of your choice. Exports can be triggered on a regular schedule or as an alert when specific criteria are met in your data. Export reporting data to share with other team members or to keep a record of key performance indicators. See [Export reporting data](https://docs.astronomer.io/astro/reporting-dash-exports) for more information.
```

This feature was always available, we just aimed to document it for release vs private preview.

File: astro/release-notes.md [hunk 3]
```diff
@@ -21,6 +21,34 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## March 5, 2024
+
+### Export data from reporting dashboards
```
Comment (lzdanski) [inline astro/release-notes.md:26]:
```suggestion
### Reporting dashboards now generally available for Enterprise tier

Previously in Private Preview, [Organization dashboards](https://docs.astronomer.io/astro/reporting-dashboard) are now available for Enterprise users to examine trends performance, use, and accrued cost for Deployments across their Organization.
```

**Review by lzdanski — COMMENTED** [review]
Some positioning changes -

File: astro/release-notes.md (hunk 3 above)
Comment (jwitz) [reply astro/release-notes.md:26]: Updated the suggestion to remove pricing tier from the text itself. When we have tags I'll add those, but I'd like to keep the content change and the product tier separate. If we end up changing the feature's tier in the future, I'd prefer to swap out a tag than rewrite the content.

**Review by lzdanski — APPROVED** [review]

File: astro/release-notes.md [hunk 4]
```diff
@@ -21,6 +21,37 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## March 5, 2024
```
Comment (lzdanski) [inline astro/release-notes.md:24]:
```suggestion
## March 7, 2024
```

**Pull request merged**